torch/csrc/utils/tensor_types.cpp

```diff
@@ -22,6 +22,7 @@ static const char* backend_to_string(const at::Backend& backend) {
     case at::Backend::HammerBlade: return "torch.hammerblade";
     case at::Backend::SparseCPU: return "torch.sparse";
     case at::Backend::SparseCUDA: return "torch.cuda.sparse";
+    case at::Backend::SparseHammerBlade: return "torch.hammerblade.sparse";
```

Comment:
Function `all_declared_types` at line 89 of this file might need modification as well...?

Reply:
That is a good point! Thanks @vb000 !